### PR TITLE
[hi] Localize content/hi/abstraction.md

### PR DESCRIPTION
**Pull request by Shubh28698**
Hi,

This PR is Hindi localization of content/hi/abstraction.md page.

ref : https://github.com/cncf/glossary/issues/270#issuecomment-998098556

/lang hi

@anubha-v-ardhan  @Garima-Negi Please review

### CODE REVIEW

File: content/hi/abstraction.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: ऐब्स्ट्रैक्शन
+status: Completed
+category: Property
+---
+
+कंप्यूटिंग के संदर्भ में, ऐब्स्ट्रैक्शन एक निरूपण है जो सेवाओं के उपभोक्ता से विशिष्टताओं को छुपाता है, एक सिस्टम को अधिक सामान्य बनाता है और इस प्रकार आसानी से समझा जाता है। आपके लैपटॉप का ऑपरेटिंग सिस्टम (OS) एक अच्छा उदाहरण है। यह आपके कंप्यूटर के काम करने के तरीके के सभी विवरणों को छुपा देता है। आपको सीपीयू, मेमोरी के बारे में और प्रोग्राम को कैसे हैंडल किया जाता है, इसके बारे में कुछ भी जानने की जरूरत नहीं है, आप बस OS को संचालित करते हैं और OS विवरण के साथ काम करता है। ये सभी विवरण OS "पर्दे" या ऐब्स्ट्रैक्शन के पीछे छिपे हुए हैं।
```
Comment (anubha-v-ardhan) [inline content/hi/abstraction.md:7]:
```suggestion
कंप्यूटिंग के संदर्भ में, ऐब्स्ट्रैक्शन एक निरूपण है जो सेवाओं के उपभोक्ता से विशिष्टताओं को छुपाता है, एक सिस्टम को अधिक सामान्य बनाता है और इस प्रकार आसानी से समझा जाता है। आपके लैपटॉप का ऑपरेटिंग सिस्टम (OS) एक अच्छा उदाहरण है। यह आपके कंप्यूटर के काम करने के तरीके के सभी विवरणों को छुपा देता है। आपको सीपीयू, मेमोरी के बारे में और प्रोग्राम को कैसे संभाला जाता है, इसके बारे में कुछ भी जानने की जरूरत नहीं है, आप सिर्फ OS को संचालित करते हैं और OS विवरणों के साथ काम करता है। ये सभी विवरण OS "पर्दे" या ऐब्स्ट्रैक्शन के पीछे छिपे हुए हैं।
```

File: content/hi/abstraction.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+सिस्टम में आमतौर पर ऐब्स्ट्रैक्शन कि गुणक परतें होती हैं। इसे डेवलपमेंट करना काफी सरल हो जाता है। प्रोग्रामिंग करते समय, डेवलपर्स एक विशेष ऐब्स्ट्रैक्शन परत के साथ संगत घटकों का निर्माण करते हैं और सभी अंतर्निहित बारीकियों के बारे में चिंता करने की ज़रूरत नहीं है जो बहुत विजातीय हो सकते हैं। यदि यह ऐब्स्ट्रैक्शन परत के साथ काम करता है, तो यह सिस्टम के साथ काम करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/abstraction.md:9]:
```suggestion
सिस्टम में आमतौर पर ऐब्स्ट्रैक्शन कि कई परतें होती हैं। इससे डेवलपमेंट काफी सरल हो जाता है। प्रोग्रामिंग करते समय, डेवलपर्स एक विशेष ऐब्स्ट्रैक्शन परत के साथ संगत घटकों का निर्माण करते हैं और सभी अंतर्निहित बारीकियों के बारे में चिंता करने की ज़रूरत नहीं होती है जो बहुत विजातीय हो सकते हैं। यदि वह ऐब्स्ट्रैक्शन परत के साथ काम करता है, तो वह सिस्टम के साथ भी काम करता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Some NITs, otherwise LGTM
Thankyou @Shubh28698

File: content/hi/abstraction.md (hunk 2 above)
Comment (Shubh28698) [reply content/hi/abstraction.md]: Thanks a lot @anubha-v-ardhan

**Comment by sayantani11** [comment]
LGTM!

**Review by Garima-Negi — COMMENTED** [review]
LGTM!

**Review by Garima-Negi — APPROVED** [review]

**Review by anubha-v-ardhan — APPROVED** [review]
Thank you @Shubh28698

**Pull request merged**